conf/server/server_full.conf

```diff
@@ -3,6 +3,11 @@
 
 # server: Contains core configuration parameters.
 server {
+    # admin_ids: SPIFFE IDs that, when present in a caller's X509-SVID, grant
+    # that caller admin privileges. The admin IDs must reside in the same
+    # trust domain as the server.
```

Comment:
> The admin IDs must reside in the same trust domain as the server.
Maybe for clarity on intention something like: "The admin IDs must reside in the same trust domain as the server, but need not have a corresponding registration entry with the server."

Reply:
Cool cool cool, will fix.